pkg/snmp/traps/listener.go

```diff
@@ -8,7 +8,6 @@ package traps
 import (
```

Comment:
👋 Could you add in PR description what was the issue causing the flakiness of the test?

And how relying on `trapsPacketsAuthErrors` instead of `receivedTrapsCount` is fixing the issue?